accessibility-checker.php

```diff
@@ -509,7 +540,22 @@ function edac_documentation_link( $rule ) {
 		return '';
 	}
 
-	return $rule['info_url'] . '?utm_source=accessibility-checker&utm_medium=software&utm_term=' . esc_attr( $rule['slug'] ) . '&utm_content=content-analysis&utm_campaign=wordpress-general&php_version=' . PHP_VERSION . '&platform=wordpress&platform_version=' . $wp_version . '&software=free&software_version=' . EDAC_VERSION . '&days_active=' . $days_active . '';
+	return add_query_arg(
+		array(
+			'utm_source'       => 'accessibility-checker',
+			'utm_medium'       => 'software',
+			'utm_term'         => esc_attr( $rule['slug'] ),
+			'utm_content'      => 'content-analysis',
+			'utm_campaign'     => 'wordpress-general',
+			'php_version'      => PHP_VERSION,
+			'platform'         => 'wordpress',
+			'platform_version' => $wp_version,
+			'software'         => 'free',
+			'software_version' => EDAC_VERSION,
+			'days_active'      => $days_active,
+		),
+		$rule['info_url']
+	);
```

Comment:
Used `add_query_arg` here to build the URL instead of having a huge string with lots of concatenation.
`add_query_arg` is slightly slower, but in this case, the code becomes a lot easier to understand and maintain so it's worth the tradeoff.